### PR TITLE
Update introduction-of-a-new-tc-member.md

### LINKED ISSUE #154: TC member onboarding checklist out-of-date
### Description
There are some updates needed to the [TC member onboarding checklist](https://github.com/eiffel-community/community/blob/master/howtos/introduction-of-a-new-tc-member.md)

For example:
- Change "Google Groups" to "Google Group" in two places
- Add a subsection for "implicit accesses" and move e.g. "Invited to the TC meetings" and "Become an Admin in the Technical Committee Tasks project ..." there, since the TC group/mailalias is (or should be) added there

### Motivation
We need a correct checklist

### Exemplification
Obvious

### Benefits
Obvious

### Possible Drawbacks
None

### PR DESCRIPTION
**Pull request by e-backmark-ericsson**
### Applicable Issues
Fixes #154 

### Description of the Change
The onboarding checklist for new TC members is out-of-date. This PR fixes that.

### Alternate Designs
None

### Benefits
Checklist up-to-date

### Possible Drawbacks
None

### Sign-off
Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Emil Bäckmark <emil.backmark@ericsson.com>

### CODE REVIEW

File: howtos/introduction-of-a-new-tc-member.md
Comment (e-backmark-ericsson) [inline howtos/introduction-of-a-new-tc-member.md:12]: This needs to be sorted out and updated before merging this PR

File: howtos/introduction-of-a-new-tc-member.md
Comment (magnusbaeck) [inline howtos/introduction-of-a-new-tc-member.md:9]:
```suggestion
- Become an Admin in the [TC-managed GitHub projects](https://github.com/orgs/eiffel-community/projects).
```

**Review by magnusbaeck — CHANGES_REQUESTED** [review]

**Comment by magnusbaeck** [comment]
While you're at it, perhaps you can add an item for the following, for which we currently have a draft issue in the TC board?

> Add item on new-TC-member checklist to enable some kind of notification (immediate and/or digest) of Dependabot alerts.

File: howtos/introduction-of-a-new-tc-member.md
Comment (e-backmark-ericsson) [reply howtos/introduction-of-a-new-tc-member.md]: It was decided on the last TC meeting that we should not continue using Google groups for the maintainers. At least not demanded from TC and not tightly connected to the GitHub repos. Updates to PROJECTS.md will happen through another issue/PR but we remove this statement here now.

**Comment by e-backmark-ericsson** [comment]
> While you're at it, perhaps you can add an item for the following, for which we currently have a draft issue in the TC board?
> 
> > Add item on new-TC-member checklist to enable some kind of notification (immediate and/or digest) of Dependabot alerts.

Yes. Done now.

**Review by magnusbaeck — APPROVED** [review]
We could add an item about adding the new member to the private TC Slack channel.

This checklist is getting awfully long. We should consider converting it into an issue template with checkboxes.

**Comment by e-backmark-ericsson** [comment]
> We could add an item about adding the new member to the private TC Slack channel.
Done

> This checklist is getting awfully long. We should consider converting it into an issue template with checkboxes.
I believe one could easily just copy this whole document into the new issue, select each bullet list and press "Add a task list"

**Review by magnusbaeck — APPROVED** [review]

**Review by t-persson — APPROVED** [review]

**Pull request merged**
